projects/adrastea/index.js

```diff
@@ -0,0 +1,52 @@
+const { Connection, PublicKey } = require('@solana/web3.js');
+const axios = require('axios');
+
+async function fetchTokenPrice(mintAddress) {
+  try {
+    const response = await axios.get(`https://api.coingecko.com/api/v3/simple/token_price/solana?contract_addresses=${mintAddress}&vs_currencies=usd`);
+    return response.data[mintAddress]?.usd || 0;
+  } catch (error) {
+    console.error('Error fetching token price:', error);
+    return 0;
+  }
+}
```

Comment:
Since the tokens are already known to Defillama, there is no need to fetch the prices